src/main/java/org/jvnet/hudson/plugins/platformlabeler/PlatformDetailsTask.java

```diff
@@ -226,15 +233,24 @@ private String checkLinux32Bit(@NonNull final String arch) {
     PREFERRED_LINUX_OS_NAMES.put("ol", "OracleServer");
     PREFERRED_LINUX_OS_NAMES.put("opensuse", "openSUSE");
     PREFERRED_LINUX_OS_NAMES.put("ubuntu", "Ubuntu");
+    PREFERRED_LINUX_OS_NAMES.put("scientific", "Scientific");
+    PREFERRED_LINUX_OS_NAMES.put("Scientific Linux", "Scientific");
+    PREFERRED_LINUX_OS_NAMES.put("Red Hat Enterprise Linux", "rhel");
```

Comment:
This is because of a difference between the os-release and redhat-release files, the former has rhel as release-id while the latter has Red Hat Enterprise Linux Server.
Because the os-release implementation was already there I opted to keep rhel as preferred release-id. As I am typing this, I see that rhel7.7 doesn't have/use lsb_release....

Reply:
That makes sense.  Left hand side from either redhat-release or os-release and it maps to the value reported in the lsb_release -a output for Distributor ID.  That seems sensible.